$NetBSD: patch-hadrian_bindist_Makefile,v 1.1 2024/04/28 05:58:57 pho Exp $

Don't run `ghc-pkg recache' while installing a binary distribution. See
also "do-install" in Makefile. This is pkgsrc-specific.

--- hadrian/bindist/Makefile.orig	2024-02-22 20:59:45.000000000 +0000
+++ hadrian/bindist/Makefile
@@ -242,7 +242,9 @@ update_package_db: install_bin install_l
 	@echo "Updating the package DB"
 	$(foreach p, $(PKG_CONFS),\
 		$(call patchpackageconf,$(shell echo $(notdir $p) | sed 's/-[0-9.]*-[0-9a-zA-Z]*\.conf//g'),$(shell echo "$p" | sed 's:\0xxx\0:   :g'),$(docdir),$(shell mk/relpath.sh "$(ActualLibsDir)" "$(docdir)"),$(shell echo $(notdir $p) | sed 's/.conf//g')))
+ifneq "$(RECACHE_PACKAGE_DB)" "NO"
 	'$(DESTDIR)$(ActualBinsDir)/$(CrossCompilePrefix)ghc-pkg' --global-package-db "$(DESTDIR)$(ActualLibsDir)/package.conf.d" recache
+endif
 
 install_mingw:
 	@echo "Installing MingGW"
